src/App.tsx

```diff
@@ -1,14 +1,48 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { FC, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [todoId, setTodoId] = useState(0);
+  const [filterType, setFilterType] = useState('all');
+  const [queryFilter, setQueryFilter] = useState('');
+
+  useEffect(() => {
+    getTodos()
+      .then((response) => {
+        setTodos(response);
+        setIsLoading(false);
+      });
```

Comment:
Let’s use async await. It gives you a simpler way to work with asynchronous promise-based code